From 7404f8f75e8df90343f7271dcee9709ffeb169f9 Mon Sep 17 00:00:00 2001
From: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
Date: Sun, 16 Dec 2018 23:05:12 +0200
Subject: [PATCH 415/741] arm64: dts: freescale: lx2160a: Inphi in112525_s03
 mdio node

commit f955d270a68e89a68e2bbc01616b6c597ba983ad from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Add Inphi retimer phyid in the mdio node, solving the probe issue
for this non-standard clause-45 device.

Signed-off-by: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
index bc77a0fd768a..79e0d078a764 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
@@ -188,11 +188,9 @@
 };
 
 &emdio2 {
-	inphi_phy: emdio2_phy@0 {
-		compatible = "ethernet-phy-ieee802.3-c45";
+	inphi_phy: ethernet-phy@0 {
+		compatible = "ethernet-phy-id0210.7440";
 		reg = <0x0>;
-		interrupts = <0 9 IRQ_TYPE_EDGE_FALLING>,
-			<0 10 IRQ_TYPE_EDGE_FALLING>;
 	};
 };
 
-- 
2.17.1

